lib/FFContextStorage.js

```diff
@@ -323,6 +322,19 @@ class FFContextStorage {
         // TODO: needed? I think not looking through @node-red/runtime/lib/nodes/context/index.js
         return []
     }
+
+    async _writeCache (scope, jsonData) {
+        const self = this
+        const path = `cache/${scope}`
+        const opts = {
+            responseType: 'json',
+            body: jsonData,
+            headers: { 'Content-Type': 'application/json;charset=utf-8' }
+        }
+        console.debug('writeCache scope ' + scope)
+        const result = await self.client.post(path, opts)
+        console.log('writeCache result', result)
```

Comment:
We should not have console.* calls in the code.

Reply:
resolved in [a0a0379](https://github.com/flowforge/flowforge-nr-persistent-context/pull/20/commits/a0a0379286f53d4d618798740174ff9d64affa04)